### PR TITLE
Update team.yml, add some logging, and fix github integration

### PR DESCRIPTION
**Pull request by KenG98**
First commit adds a "location" variable to the env sample, which is TESTING or PRODUCTION.

Next I updated the team.yml to reflect our current membership (removing old members and adding new ones).

Then I fixed the github.rb script, fixing the bug introduced earlier.

I verified that it works by running it with the TESTING flag, which printed exactly who needs to be removed, added, and whose permissions need to be updated.

### CODE REVIEW

**Comment by KenG98** [comment]
I ran squadra in production, here's the output

```
Squadra is beginning...
~~~ Validating team.yml ~~~
~~~ Finished validating team.yml ~~~
~~~ Starting github team update ~~~
a--hoang is a member, no change necessary.
alexfarra is a member, no change necessary.
austinnegron is a member, no change necessary.
bianca-sparxs is a member, no change necessary.
*** charlesma4 is admin but should be member.
Calling Github API, changing perms for user: charlesma4
emampons is a member, no change necessary.
gabriellasroman is a member, no change necessary.
jjuncho is a member, no change necessary.
jli12 is a member, no change necessary.
keng98 is a admin, no change necessary.
lisaqvu is a member, no change necessary.
*** lkwatson is admin but should be member.
Calling Github API, changing perms for user: lkwatson
maridematte is a admin, no change necessary.
mhendrick98 is a admin, no change necessary.
momolin78 is a member, no change necessary.
nampham148 is a member, no change necessary.
noahnaiman is a member, no change necessary.
*** puzzledsean should be removed from the organization.
Calling Github API, removing user: puzzledsean
rooday is a member, no change necessary.
sarah256 is a member, no change necessary.
sidpremkumar is a member, no change necessary.
smroh17 is a member, no change necessary.
warren-partridge is a member, no change necessary.
*** ibrahims123457 should be added to the organization as a member.
Calling Github API, updating user: ibrahims123457
*** sarahgreisdorf should be added to the organization as a admin.
Calling Github API, updating user: sarahgreisdorf
*** victoriakayola should be added to the organization as a member.
Calling Github API, updating user: victoriakayola
*** benlague should be added to the organization as a member.
Calling Github API, updating user: benlague
*** jennkimerson should be added to the organization as a member.
Calling Github API, updating user: jennkimerson
*** rogerramesh should be added to the organization as a member.
Calling Github API, updating user: rogerramesh
*** gracejpeg should be added to the organization as a member.
Calling Github API, updating user: gracejpeg
~~~ Finished updating github membership ~~~
~~~ Starting mailgun team update ~~~
Deleting mailgun routes
Deleting mailgun routes
Deleting mailgun routes
Deleting mailgun routes
Deleting mailgun routes
Deleting mailgun routes
Routes flushed, rebuilding
Creating new route with description: General catch-all bostonhacks emails
Added email route contact@bostonhacks.io, mailgun response: Route has been created
Creating new route with description: Mail, used for mailchimp login
Added email route mail@bostonhacks.io, mailgun response: Route has been created
Creating new route with description: Resumes for bostonhacks applicants
Added email route resumes@bostonhacks.io, mailgun response: Route has been created
Creating new route with description: Receipts and record emails
Added email route records@bostonhacks.io, mailgun response: Route has been created
Creating new route with description: Daily Free Press Team
Added email route dfp@bostonhacks.io, mailgun response: Route has been created
Creating new route with description: Sponsor Team
Added email route sponsor@bostonhacks.io, mailgun response: Route has been created
~~~ Finished updating mailgun routes ~~~
Squadra is finished.
```

I confirmed visually that the memberships on github updated, and I confirmed on mailgun that the routes were updated.

Running the script again shows the changes took effect (except for added users, but that is to be expected, they haven't yet accepted their invites).

File: github.rb [hunk 1]
```diff
@@ -1,66 +1,81 @@
...
 client = Octokit::Client.new(
     :login => ENV['GITHUB_USERNAME'],
     :password => ENV['GITHUB_ACCESS_TOKEN'])
 
+# current members of the github organization
 current_members = client.organization_members('bostonhacks')
-current_permissions = {}
-# The permissions on this repo are the current permissions
+
+# iterate through all members of the organization to find:
```
Comment (a--hoang) [inline github.rb]: ty for adding comments hahah i am so sorry

File: team.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ members:
     -   name: Andrew Hoang
         github:
             username: a--hoang
-            role: admin
+            role: member
```
Comment (a--hoang) [inline team.yml]: :')

**Comment by a--hoang** [comment]
10/10 would merge

**Comment by KenG98** [comment]
Big thanks to @lkwatson @a--hoang for the reviews and hard work on squadra

**Comment by lkwatson** [comment]
Nah big thanks to you @KenG98 for fixing this

**Pull request merged**